templates/store/storeConfig.ts

```diff
@@ -1,6 +1,6 @@
 import { createStore, applyMiddleware, compose } from 'redux';
 import logger from 'redux-logger';
-{{#ifEquals middleware supportedMiddlewares.reduxObservable}}
+{{#ifEquals middleware 'reduxObservable'}}
```

Comment:
why like that? imho it would be safer to keep strings as const and import them from config file